examples/2D_jet_in_crossflow/case_2D_jicf.py

```
@@ -0,0 +1,115 @@
+                "bc_y%beg": -17,
+                "bc_y%end": -3,
+                "num_bc_patches": 0,
+                #Formatted Database File Structures
+                "format": 1,
+                "precision": 2,
+                "prim_vars_wrt": "T",
+                "parallel_io": "T",
+                #Patch 1: Free stream
+                "patch_icpp(1)%geometry": 3,
+                "patch_icpp(1)%x_centroid": 0.0,
+                "patch_icpp(1)%y_centroid":50*djet,
```

Ruff (0.15.9)

[error] Undefined name `djet`

(F821)